pypd/mixins.py

```diff
@@ -45,6 +46,13 @@ def __init__(self, api_key=None, base_url=None):
             from pypd import base_url
             self.base_url = base_url
 
+        if proxies:
```

Comment:
I think this is my only comment for your code - you could probably get as concise here if you used the if statement like:
```python
if not proxies:
    from pypd import proxies
self.proxies = proxies
```